### PR TITLE
Add DG portion for ViewCommand and Russell's Contribution to Meet Command

### PR DESCRIPTION
**Pull request by russelltankaimin**
[See Page Here!](https://ay2223s2-cs2103t-w14-2.github.io/tp/DeveloperGuide.html)

### CODE REVIEW

**Comment by russelltankaimin** [comment]
Putting this PR to point to master was intentional due to the confusion in the workflow with regards to organisation repo's branch.

**Comment by russelltankaimin** [comment]
@zichen-3974 Any idea how to fix this issue on arrow misalignment at the bottom?
Screenshot 2023-03-22 at 10 48 30 AM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -558,6 +601,29 @@ The `sort` command allows users to arrange their contacts in the order they desi
...
+<div markdown="span" class="alert alert-warning">
+    :construction: Integration works in the process.
+</div>
+
+<div markdown="span" class="alert alert-info">
+
+:information_source: **Sub-Commands** <br>
+The `meet` command has 3 different sub-commands : `eat`, `study` and generally `meet`.
+However, we will be referring to all 3 commands generally as `meet`. All 3 commands only differ in the location
+recommendations as certain locations are only appropriate for certain activities.
```
Comment (mynameizzhafeez) [inline docs/DeveloperGuide.md:615]: This could be elaborated further but it is fine.

File: docs/images/ViewParserSequenceDiagram.png
Comment (mynameizzhafeez) [inline docs/images/ViewParserSequenceDiagram.png:1]: Yes now it looks very nice :)

File: docs/images/ViewActivityDiagram.png
Comment (mynameizzhafeez) [inline docs/images/ViewActivityDiagram.png:1]: Could probably follow our lecture notation for if nodes: the `[True]` should probably be `contactIndex present` for example.

**Review by mynameizzhafeez — APPROVED** [review]
LGTM.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -510,9 +517,45 @@ The Sequence Diagram below illustrates the interactions within the `Logic` compo
...
+<div markdown="span" class="alert alert-info">
+
+:information_source: **Command Formats** <br>
+- `view` : Displays user's profile on the display panel. <br>
+- `view n/XYZ` : Display XYZ's profile on the display panel. <br>
+- `view <INDEX>` : Display the contact's whose `ContactIndex` is `INDEX` on the display panel.
+</div>
+
+**Parsing the inputs** - When the user enters the input, the `ViewCommandParser` will check if the arguments are empty first.
+If it is not empty, then `ViewCommandParser` will extract try to find tokens that were prefixed `/n`. If a name is not present, it will search for an index (of `int` type) instead.
```
[the remark this answers is not in the document]
Comment (russelltankaimin) [reply docs/DeveloperGuide.md:544]: OK!

**Pull request merged**